### DISCUSSION TITLE
What are the requirements for CustomTkinter?

### DISCUSSION THREAD

**Discussion by stepanzh** [TomSchimansky/CustomTkinter#1260] [Q&A] [answered]
First of all, great looking package! Thanks!

My question is what are OS and Python requirements for CustomTkinter?

I'm working on a GUI app, that should work on both Windows 7 and Windows 10. For this, I'm using embeddable version of [Python 3.8.10](https://www.python.org/downloads/release/python-3810/) with tkinter and third-party libraries. Everything works ok, and now I'm looking for ways to style the app. CustomTkinter looks very promising for me. Would it work on Windows 7 with Python 3.8.10?

**Comment by Akascape** [comment, marked as answer]
Requirements for customtkinter:

**Supported OS:** Windows, Linux and Mac
**minimum python version:** 3.7+

When it comes to windows, win 10 and win 11 are working properly.

But it is not tested on windows 7, so you have to manually check it.
